holidays/holiday_base.py

```diff
@@ -230,28 +241,31 @@ def __init__(
         :param state:
             *deprecated* use subdiv instead.
 
-        :param expand:
-            Whether the entire year is calculated when one date from that year
-            is requested.
-
-        :param observed:
-            Whether to include the dates when public holiday are observed
-            (e.g. a holiday falling on a Sunday being observed the
-            following Monday). This doesn't work for all countries.
+        :param language:
+            The language which the returned holiday names will be translated
+            into. It must be an ISO 2-digit language code. If the language
+            translation is not supported the original holiday names will be
+            used.
```

Comment:
@mborsetti at the moment the library has a mix of english and local names for holidays, so probably that should be preserved for backwards compatibility.

Reply:
First of all many thanks for the review, @mborsetti !

> I propose instead that we standardize the language returned when no language code is present.

I believe this approach is what Roman has originally implemented: use `en` as a default. I don't mind to re-work the logic back. My reasoning behind the `language = None` option is compatibility (learned the importance of this after many reviews by @dr-prodigy :) ). 
On the other hand it's kind of better UX to have holiday names in the user's language by default (instead of English).

>I further propose that such standard be that this project behaves the same way as gettext, i.e. the language returned is based on the "current global domain, language, and locale directory". However, if no match is found, then en language will be returned.

That's an interesting idea. I'll look into it.

I admit that sometimes (quite often) it's impossible to go with the right approach due to compatibility reasons.